### PR TITLE
use color boxes for lanyards

### PR DESCRIPTION
**Pull request by dzoladz**
First attempt to improve the lanyard info. I've added a spot the new zebra lanyards.

Screen Shot 2022-02-27 at 8 42 59 PM

### CODE REVIEW

File: conduct/index.html [hunk 1]
```diff
@@ -281,12 +281,40 @@ <h2 id="photography">Photography Policy</h2>
...
-                        <li><span style="background-color:#000; color: #ffff16;font-weight: 700;">Yellow</span>: Ask before photographing</li>
-                        <li><span style="color: #710d0d;font-weight: 700;">Red</span>: Photographs never okay, don’t ask</li>
-                    </ul>
+                    <div class="container">
+                      <div class="row lanyard-row">
+                        <div class="col-md-3 lanyard-colorbox lanyard-green">
+                          Green
+                        </div>
+                        <div class="col-md-9">
+                          Printed with {code4lib}: Photographs always okay, as this represents expressly-given permission beforehand
```
Comment (phette23) [inline conduct/index.html:290]: I don't understand "printed with {code4lib}" here? Are you saying that the green lanyards will say "Code4Lib" but none of the other ones will? And why is it wrapped in curly braces?

File: conduct/index.html [hunk 2]
```diff
@@ -281,12 +281,40 @@ <h2 id="photography">Photography Policy</h2>
...
+                        <div class="col-md-3 lanyard-colorbox lanyard-red">
+                          Red
+                        </div>
+                        <div class="col-md-9">
+                          Photographs never okay, don’t ask
+                        </div>
+                      </div>
+                      <div class="row lanyard-row">
+                        <div class="col-md-3 lanyard-colorbox lanyard-zebra">
+                          Zebra
```
Comment (phette23) [inline conduct/index.html:311]: I listened to this section in a screenreader and it read great, very clear, but I think "Zebra-striped" is a slightly better way of describing the CSV lanyards.

**Review by phette23 — COMMENTED** [review]
Added a couple comments to address. Looks great! A big improvement for sure.

File: conduct/index.html (hunk 1 above)
Comment (dzoladz) [reply conduct/index.html:290]: tbh, I just copy/pasted all of the previous text. I'm fine with removing the entire `Printed with {code4lib}:` part. It's unnecessary.

File: conduct/index.html (hunk 2 above)
Comment (dzoladz) [reply conduct/index.html:311]: I'll make this change tonight.

**Pull request merged**